src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -93,7 +93,7 @@ public static void TestResolveInvalidPaths()
         [Fact]
         public static void TestResolveInvalidPath()
         {
-            Assert.Throws<System.Net.WebException>(() => XmlReader.Create("ftp://host.invalid"));
+            Assert.ThrowsAny<Exception>(() => XmlReader.Create("ftp://host.invalid"));
```

Comment:
SocketsHttpHandler doesn't handle non-HTTP uri's.  So ftp:// scheme wouldn't be handled at all. So, maybe this test should be switched to use "http" scheme (even though it is testing for an invalid path).

Reply:
Fixed.